### DISCUSSION TITLE
vtk-data not exist any more in version 0.37.0,some example code need it

### DISCUSSION THREAD

**Discussion by chinvqin** [pyvista/pyvista#3833] [Q&A] [answered]
with example **3D Earth and Celestial Bodies**
need some resources as below
   # Load planets
    mercury = examples.planets.load_mercury(radius=2439.0)
    venus = examples.planets.load_venus(radius=6052.0)
    earth = examples.planets.load_earth(radius=6378.1)
    mars = examples.planets.load_mars(radius=3397.2)
    jupiter = examples.planets.load_jupiter(radius=71492.0)
    saturn = examples.planets.load_saturn(radius=60268.0)
but in github folder **vtk-data/** not exist any more in version 0.37.0
ConnectionError: HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /pyvista/vtk-data/raw/master/Data/solar_textures/mercury.jpg
how can I get these resources? thanks for your notice

**Comment by MatthewFlamm** [comment, marked as answer]
Do you have an environment variable `PYVISTA_VTK_DATA` set?  See https://docs.pyvista.org/extras/vtk_data.html.

**Reply by MatthewFlamm** [reply]
If you do, and it is pointing to the correct folder, you need to keep that repo up to date.

**Reply by chinvqin** [reply]
got it, I'll try
thank you so much
